todoman/formatters.py

```diff
@@ -82,7 +82,6 @@ def detailed(self, todo):
             extra_rows += self._columnize('Description', todo.description)
         if todo.location:
             extra_rows += self._columnize('Location', todo.location)
-
```

Comment:
please reply, I think I do all requested changes Now, what change I should do because when I run this manually it work accordingly as we want.